apps/console/src/public/deployment.config.json

```
@@ -37,7 +37,7 @@
         "clockTolerance": 300,
         "responseMode": "query",
         "scope": [ "SYSTEM", "profile", "openid" ],
-        "serverOrigin": "https://localhost:9443",
+        "serverOrigin": "https://localhost:9444",
```

Let's revert these values to default :)

```suggestion
        "serverOrigin": "https://localhost:9443",
```

Check other places as well.

---

Resolved by db85c36f8b0d626692920504659aa1da0a48bc01